austation/code/game/objects/items/tanks/tank_types.dm

```diff
@@ -0,0 +1,118 @@
+/////////////////////  BASIC TANK  /////////////////////
+
+
+//  New Feature, Alt-Clicking a tank resets it to the original 17kPa pressure, not that you'll probably ever use this often.
+//  Also supports swapping between multiple modes from a list.
+//  If by any chance somebody adds another tank with multiple modes, it can use this system by just plugging in a new list into var/list/modes()
+/obj/item/tank/internals
+	var/list/modes = list(TANK_DEFAULT_RELEASE_PRESSURE)  //  MUST be ordered numerically from lowest to highest!
+
+/obj/item/tank/internals/AltClick(mob/user)
+	. = ..()
+	if(!length(modes))  //  Something weird happened
+		distribute_pressure = TANK_DEFAULT_RELEASE_PRESSURE  //  Probably for the best?
+		return
+	if(length(modes) == 1)
+		distribute_pressure = modes[1]
+		to_chat(user, "<span class='notice'> You quickly adjust \the [src] to release [distribute_pressure]kPa.</span>")
```

Comment:
there's also an extra space sitting in this string

Reply:
Changed